sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,195 @@
+---
+id: high-contrast-document
+title: High Contrast in FAST
+sidebar_label: High Contrast
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/design-systems/high-contrast.md
+---
+
+### Styling components using forced-colors.
```

Comment:
The rest of our docs have L2 headers at the root. So, let's switch this and the other headers at this same level over.